### PR TITLE
added carousel to member section

### LINKED ISSUE #25: Add a carousel to the members section

### PR DESCRIPTION
**Pull request by debck**
Fixes #25

### CODE REVIEW

**Comment by debck** [comment]
@knrt10  Plz help why the build is failing.

File: assets/js/carousel.js [hunk 1]
```diff
@@ -0,0 +1,31 @@
+$(document).ready(function(){
+  var owl = $('.owl-carousel');
+  owl.owlCarousel({
+    loop:true,
```
Comment (knrt10) [inline assets/js/carousel.js]: add 
```
autoplay: true,
autoplayTimeout: 3000,
```

Comment (knrt10) [reply assets/js/carousel.js]: move this whole file below `index.html` in script tag then test won't fail

**Comment by knrt10** [comment]
@debck good work

**Pull request merged**